server/images.go

```diff
@@ -738,6 +738,8 @@ func PullModel(name string, regOpts *RegistryOptions, fn func(api.ProgressRespon
 	return nil
 }
 
+var errModelNotFound = fmt.Errorf("model not found")
```

Comment:
Is there a need to create an error var? I don't see anything using it other than http.StatusNotFound